pkg/cmd/clustertask/delete_test.go

```diff
@@ -296,6 +296,15 @@ func TestClusterTaskDelete(t *testing.T) {
 			wantError:   true,
 			want:        "must provide ClusterTask name(s) or use --all flag with delete",
 		},
+		{
+			name:        "Delete the ClusterTask present and give error for non-existent ClusterTask",
+			command:     []string{"delete", "nonexistent", "tomatoes2"},
+			dynamic:     seeds[5].dynamicClient,
+			input:       seeds[5].pipelineClient,
+			inputStream: nil,
+			wantError:   true,
+			want:        "failed to delete ClusterTask \"nonexistent\": clustertasks.tekton.dev \"nonexistent\" not found",
```

Comment:
You are asserting the error here, but should also assert the other tasks that got deleted